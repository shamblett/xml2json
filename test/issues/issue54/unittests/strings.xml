<?xml version="1.0" encoding="utf-8"?>
<strings>
  This text should be ignored
  <script><![CDATA[<script>alert("YES")</script>]]></script>
  <string>Zooi</string>
  <string>Tab	Text</string>
  <string>Quote "test"</string>
  <string>Backslash \ test</string>
  <string>Quote " en \ test</string>
  <string>"Forsooth! It 'tis as I thought!"</string>
  <string>"Begin/end quote"</string>
  <escape>Line one (één in Dutch)
Line two:	tab
He said: "Much fun with a €"
More unicode&#8224;</escape>
   <poetry>JABBERWOCKY

Lewis Carroll

(from Through the Looking-Glass and What Alice Found There, 1872)
`Twas brillig, and the slithy toves
  Did gyre and gimble in the wabe:
All mimsy were the borogoves,
  And the mome raths outgrabe.

 

"Beware the Jabberwock, my son!
  The jaws that bite, the claws that catch!
Beware the Jubjub bird, and shun
  The frumious Bandersnatch!"
He took his vorpal sword in hand:
  Long time the manxome foe he sought --
So rested he by the Tumtum tree,
  And stood awhile in thought.
And, as in uffish thought he stood,
  The Jabberwock, with eyes of flame,
Came whiffling through the tulgey wood,
  And burbled as it came!
One, two! One, two! And through and through
  The vorpal blade went snicker-snack!
He left it dead, and with its head
  He went galumphing back.
"And, has thou slain the Jabberwock?
  Come to my arms, my beamish boy!
O frabjous day! Callooh! Callay!'
  He chortled in his joy.



`Twas brillig, and the slithy toves
  Did gyre and gimble in the wabe;
All mimsy were the borogoves,
  And the mome raths outgrabe.</poetry>
</strings>
