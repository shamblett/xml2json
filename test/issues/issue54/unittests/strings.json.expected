{"strings":{"script":"<script>alert(\"YES\")</script>","string":"Zooi","string":"Tab\tText","string":"Quote \"test\"","string":"Backslash \\ test","string":"Quote \" en \\ test","string":"\"Forsooth! It 'tis as I thought!\"","string":"\"Begin/end quote\"","escape":"Line one (één in Dutch)\nLine two:\ttab\nHe said: \"Much fun with a €\"\nMore unicode†","poetry":"JABBERWOCKY\n\nLewis Carroll\n\n(from Through the Looking-Glass and What Alice Found There, 1872)\n`Twas brillig, and the slithy toves\n  Did gyre and gimble in the wabe:\nAll mimsy were the borogoves,\n  And the mome raths outgrabe.\n\n \n\n\"Beware the Jabberwock, my son!\n  The jaws that bite, the claws that catch!\nBeware the Jubjub bird, and shun\n  The frumious Bandersnatch!\"\nHe took his vorpal sword in hand:\n  Long time the manxome foe he sought --\nSo rested he by the Tumtum tree,\n  And stood awhile in thought.\nAnd, as in uffish thought he stood,\n  The Jabberwock, with eyes of flame,\nCame whiffling through the tulgey wood,\n  And burbled as it came!\nOne, two! One, two! And through and through\n  The vorpal blade went snicker-snack!\nHe left it dead, and with its head\n  He went galumphing back.\n\"And, has thou slain the Jabberwock?\n  Come to my arms, my beamish boy!\nO frabjous day! Callooh! Callay!'\n  He chortled in his joy.\n\n\n\n`Twas brillig, and the slithy toves\n  Did gyre and gimble in the wabe;\nAll mimsy were the borogoves,\n  And the mome raths outgrabe."}}
